images/base/Dockerfile

```diff
@@ -128,7 +128,7 @@ RUN git clone --filter=tree:0 "${CONTAINERD_CLONE_URL}" /containerd \
 # stage for building runc
 FROM go-build as build-runc
 ARG TARGETARCH GO_VERSION
-ARG RUNC_VERSION="v1.1.7"
+ARG RUNC_VERSION="v1.1.5"
```

Comment:
per recent discussion: it's acceptable to have an older runc than the vendored copy in kubelet, but being ahead can cause problems on cgroups v1

so we're going to an older version, this is still new enough to have the recent CVE patches.

thankfully runc version is independent of containerd version in the base image now, so we can avoid rolling back containerd